product-service/src/handlers/get-products.ts

```diff
@@ -0,0 +1,24 @@
+import { HEADERS } from '../shared/constants';
+import ProductService from '../services/product-service';
+import { Response } from '../interfaces';
+
+export const getProducts = async ():Promise<Response> => {
+  const productService = new ProductService();
+
+  try {
+    const products = await productService.getProducts();
+
+    return {
+      statusCode: 200,
+      body: JSON.stringify(products),
+      headers: HEADERS,
+    };
+  } catch (error) {
+    return {
+      statusCode: 400,
```

Comment:
Not a big deal, but I think 500 statusCode is more accurate here because 4** codes are client ones. In this case, we are going to get to the "catch" block if something is wrong with getting products (server error, not a client one).

Reply:
hi
i was going to change it when we have some DB